app/views/bookings/index.html.erb

```diff
@@ -11,10 +11,23 @@
 </p>
 
 <h2>The full list of bookings:</h2>
+
 <div class='bookings'>
   <% if @bookings.any? %>
     <%= render partial: 'bookings/booking', collection: @bookings %>
   <% else %>
     There is no booking yet.
   <% end %>
 </div>
+
+
+<div class="pagination-options">
+  <%= form_tag bookings_path, method: :get do %>
+    <%= select_tag :per_page, options_for_select([4, 6, 8, 12, 20], params[:per_page]), onchange: "this.form.submit()" %>
+  <% end %>
+</div>
+
+<div class="flickr_pagination">
+  
+    <%= will_paginate :container => true %>
```

Comment:
Csomószor fogsz találkozni Rails-es forrásokkal, akik olyan formában adnak meg hash-eket, mint ahogy fent is írtad.
```ruby
{ :kulcs => 'érték' }
```
Ez viszont teljesen egyenértékű az alábbival. (Ez az újabb és a preferált syntax.)
```ruby
{ kulcs: 'érték' }
```